webpack.config.cjs

```diff
@@ -204,6 +204,9 @@ const sharedConfig = {
           output: {
             comments: /translators:/i,
           },
+          mangle: {
+            reserved: ['__', '_x', '_n', '_nx']
```

Comment:
I also tried this today, does it have any effect?

Reply:
Not really, no. Or at least not the desired effect. Because `usedExports` does tree shaking _and_ mangling of exports, so we'll get output like `Object(i18n["a" /* __ */])('Direction', 'web-stories')` that is then passed to Terser.

That would actually be fine, since the i18n tooling recognizes this format.

But then Terser removes the `/* __ */` comments and does all sorts of mangling and we'll end up with something more like `Object(c.a)("Direction","web-stories")`